server/core/src/main/java/org/infinispan/server/core/configuration/ProtocolServerConfigurationBuilder.java

```diff
@@ -109,6 +111,10 @@ public SslConfigurationBuilder ssl() {
       return ssl;
    }
 
+   public IpFilterConfigurationBuilder ipRules() {
```

Comment:
method's name "should" be `ipFilter` IMO
```suggestion
   public IpFilterConfigurationBuilder<T, S> ipFilter() { {
```

should `ProtocolServerConfigurationChildBuilder` have this method as well?